---
output: pdf_document
papersize: letter
geometry: margin=0.7in
fontsize: 11pt
numbersections: true
linkReferences: true
nameInLink: false
colorlinks: false
header-includes:
  - \usepackage{fontspec}
  - \usepackage{fontawesome}
---

######################################################################################################################################################################


\title{\vspace{-3.5em}\textbf{Arvin Zaker}\vspace{-3.5em}}
\maketitle

\centering{\href{mailto:arvzaker@student.ubc.ca}{\underline{\textsc{\faEnvelopeO} arvzaker@student.ubc.ca}} |
\href{https://github.com/ArvinZaker}{\underline{\textsc{\faGithub} github.com/ArvinZaker}} }

\flushleft

######################################################################################################################################################################


\vspace{-0.5em}

# Educational Background

**Honours Bachelor of Science - Translational and Molecular Medicine |  (September 2019 - April 2023)** 

_University of Ottawa, Ottawa, ON_

- CGPA: 9.98/10
- Major Research Project Supervisor: Dr. Arvind Mer
- Major Research Project Title: Comparative analysis of drug response metrics in patient-derived xenografts


######################################################################################################################################################################

\vspace{-0.5em}

# Honours, Scholarships and Awards

**NSERC - Undergraduate Student Research Awards (USRA) | (May 2023)**

_University of Ottawa, Ottawa, ON_

- Competitive merit-based award. Selection based on academic performance, quality of the research proposal, and the research's potential impact.
- One of 88 awards available at the University of Ottawa.
- Valued at $7500 for 3 months

\vspace{0.25em}


**Work in Biomedical Research Scholarship | (May 2022)** 

_University of Ottawa, Ottawa, ON_

- Competitive merit-based award. Selection based on academic performance and the quality of the research proposal.
- Under 50% success rate.
- Valued at $3500 for 3 months

\vspace{0.25em}

**TMM Summer Training program | (May 2022)** 

_University of Ottawa, Ottawa, ON_

- Competitive merit-based award. Selection based on based on academic performance and the quality of the research proposal.
- Valued at $8900 for 3 months.

\vspace{0.25em}


**Undergraduate Research Opportunities (UROP) | (December 2020)** 

_University of Ottawa, Ottawa, ON_

- Competitive merit-based award. Selection based on academic performance and the quality of the research proposal.
- Valued at $1000 for 4 months.

\vspace{0.25em}

**Dean's Honour List & Merit Scholarship | (September 2019 - April 2023)** 

_University of Ottawa, Ottawa, ON_

- Merit-based award. Selection based on grade point average (GPA) in post-secondary studies.
- Valued at $1000 per term (awarded every term).

\vspace{0.25em}

\pagebreak


######################################################################################################################################################################

\vspace{-0.5em}

# Skills

+ Experienced in bioinformatics  data analysis using R and Python programming languages
+ Proficiency in developing reproducible bioinformatics pipelines in Unix systems with a set of languages such as Bash, Nextflow and Nix.
+ Expertise in Multi-omics data analysis, including genomics, epigenomics, and transcriptomics.
+ Proficient in bioinformatics programming using R and Python programming languages
+ Expert in developing reproducible bioinformatics workflows across Unix and high-performance computing environments.
+ Advanced machine learning model development in cancer genomics and precision medicine. 
+ Proficient in data visualization and statistical interpretation of complex biological datasets.


######################################################################################################################################################################


\vspace{-0.5em}

# Research Experience

**Research Assistant - Epigenomics \hfill March 2024 - Present**

**Supervisor:** Dr. Arvind Mer (Primary Supervisor) and 

**Contribution:**

+ Collaborators: Dr. Ben Tsang, Dr. Gilda Stefanelli, and Dr. Jean-Francois Couture.
+ Perform epigenomic analysis of clinical exosomal DNA samples to identify differentially methylated regions and pathways.
+ Created and optimized Cut&Run epigenomic analysis pipeline to explore the developmental impacts of neurological conditions in mice.
+ Integrated and aggregated multi-omics data from 44 studies and databases.
+ Performing external validation of the results on public datasets.

\vspace{0.25em}


**Research Assistant - Machine learning \hfill May 2023 - February 2024**

**Supervisor:** Dr. Arvind Mer 

**Contribution:** 

+ Developed machine-learning-based drug response metrics to predict the probability of response to anti-cancer medication.
+ Compared multiple feature selection methods for drug-response prediction machine-learning models to identify the most suitable methods for translatable clinical models. 
+ Applied the machine learning model to predict drug response to chemotherapies.
+ Visualized the performance differential between machine learning models.

\vspace{0.25em}

**Undergraduate Thesis \hfill September 2022 - April 2023**

**Supervisor:** Dr. Arvind Mer

**Title:** Comparative Analysis of Drug Response Metrics for Patient-Derived Xenograft

**Contribution:** 

+ Performed literature review to identify common methods of quantifying drug response in patient-derived xenografts (PDX).
+ Developed xeMetron package to calculate 250 existing and novel metrics for measuring the drug response in PDX.
+ Performed biomarker discovery to identify transcriptional markers of drug sensitivity in chemotherapies.
+ Created machine learning algorithms to predict drug response in clinical datasets.

\vspace{0.25em}

**Summer Research Assistant \hfill January 2022 - August 2022** 

**Supervisor:** Dr. Arvind Mer (Primary Supervisor) and Dr. Damien D'Amours

**Title:** Large-scale phenogenomic analysis of human cancers uncovers frequent alterations affecting SMC5/6 complex components in breast cancer

**Contribution:** 

+ Aggregated and combined data from 199 cancer studies to analyze the mutational effects of SMC5/6 DNA repair complex in a pan-cancer cohort.
+ Utilized R programming language to analyze the functional impact of SMC5/6 mutation using transcriptomic data.
+ Performed cancer phenotype and survival analysis to study the clinical implication of SMC5/6.

\vspace{0.25em}

**Student Research Assistant \hfill September 2021 - April 2022**

**Supervisor:** Dr. Adam Rudner

**Title:** Discovery and Characterization of Phage Arzan and Khorshid

**Contribution:** 

+ Performed environmental sampling, phage extraction, and phage amplification assays.
+ Conducted DNA extraction and restriction digest analysis via gel electrophoresis.
+ Discovered and annotated the genome of two novel phages, Arzan and Khorshid.

\vspace{0.25em}

**UROP Research Assistant \hfill November 2020 - July 2022**

**Supervisor:** Dr. Jean-Marc Renaud

**Title:** Role of Adenosine Receptors on Mice Skeletal Muscle Fatigue Kinetics.

**Contribution:** 

+ Designed experiments and set up research equipment for data collection.
+ Performed experiments on mouse muscles using electrophysiological equipment.
+ Performed antibody staining, fluorescent imaging, and analysis of muscle tissue in the Imiris program.

######################################################################################################################################################################

\vspace{-0.5em}

# Publications, Presentations, and Abstracts

**Publications**

Asare-Werehene, M., **Zaker, A.**, Tripathi, S., Communal, L., Carmona, E., Mes-Masson, A., Tsang, B., and Mer, A. (2024). Spotlight on Nuclear PD-L1 in Ovarian Cancer Chemoresistance: Hidden but Mighty. _Cancer Research_ (Submitted manuscript in December 2024, currently under review). 

\vspace{0.25em}

Grégoire, S., Grégoire, J., Yang, Y., Capitani, S., Joshi, M., Sarvan, S., **Zaker, A.**, Ning, Z., Figeys, D., Ulrich, K., Brunzelle, J. S., Mer, A., & Couture, J.-F. (2024). Structural insights into an atypical histone binding mechanism by a PHD finger. _Structure_, 32(9), 1498-1506.e4. \href{https://doi.org/10.1016/j.str.2024.06.017}{\underline{\textsc{doi: 10.1016/j.str.2024.06.017}}}

\vspace{0.25em}

Wu, G., **Zaker, A.**, Ebrahimi, A., Tripathi, S., & Mer, A. S. (2024). Text-mining-based feature selection for anticancer drug response prediction. _Bioinformatics Advances_, 4(1), vbae047. \href{https://doi.org/10.1093/bioadv/vbae047}{\underline{\textsc{doi: 10.1093/bioadv/vbae047}}}

\vspace{0.25em}

Nasser, F., Gaudreau, A., Lubega, S., **Zaker, A.**, Xia, X., Mer, A. S., & D’Costa, V. M. (2024). Characterization of the diversity of type IV secretion system-encoding plasmids in Acinetobacter. _Emerging Microbes & Infections_, 13(1), 2320929. \href{https://doi.org/10.1080/22221751.2024.2320929}{\underline{\textsc{doi: 10.1080/22221751.2024.2320929}}}

\vspace{0.25em}

Roy, S., **Zaker, A.**, Mer, A., & D’Amours, D. (2023). Large-scale phenogenomic analysis of human cancers uncovers frequent alterations affecting SMC5/6 complex components in breast cancer. _NAR Cancer_, 5(3), zcad047. \href{https://doi.org/10.1093/narcan/zcad047}{\underline{\textsc{doi: 10.1093/narcan/zcad047}}}

\vspace{0.25em}

**Posters and presentations at scientific meetings**

Roy, S., **Zaker, A.**, Mer, A., D'Amours, D. (2023). Large-scale phenogenomic analysis of human cancers uncovers frequent alterations affecting SMC5/6 complex components in breast cancer. _Canadian Society for Molecular Biosciences (CSMB) International Conference_. Ottawa, ON. (poster and presentation)

\vspace{0.25em}

**Zaker, A.**, Mer, A. (2022). Comparative analysis of drug response metrics in patient-derived xenografts. _4th Annual University of Ottawa Faculty of Medicine Research Day._
Ottawa, ON. (oral presentation)

\vspace{0.25em}

Friesen, R., Algharbi S., **Zaker, A.**, Mahdi, O., Gao, R., Ferri, O., Li, J.,
Wang, SY., HeffernanC., Featherstone, A., Radar, A., Gandelman, M., Chander N.,
Bancud, SE., Rege, I., Shriraam, R., Jung, D., Karunakaran, G., Sarakbi, R.,
Znamenski, E.,  Ristovski, M., Freitas, JD., McCarthy, L., Williams, EC.,
D'Ambrosio, L., Chan, K., Wheaton, K., Rudner, AD. (2022). 
Investigating
Nucleotide-binding Proteins in Bacteriophage JohnDoe. 
_SEA Symposium_. Pittsburgh, PA. (abstract & presentation).

\vspace{0.25em}

Featherstone, A., Radar, A., Salama, A., Tiukuvaara, S., Ferri, O., **Zaker, A.**, Algharbi, S., Mahdi, O., Setia, G., Friesen, R., Wang, GY., Li, J., Gao, R., Jung, D., Karunakaran, G., Znamenski, E., Ristovski, M., Sarakbi, R., Freitas, JD., McCarthy, L., Williams, EC., D'Ambrosio, L., Wheaton, K., Rudner AD. (2022).
Give Us a Millet of Your Time: Dehusking the Singleton Bacteriophage Arzan.
_SEA Symposium_. Pittsburgh, PA. (abstract & presentation)

\vspace{0.25em}

**Zaker, A.**, McCarthy, L., Jung, D., Karunakaran, G., Rudner, A. (2022). Discovery and characterization of bacteriophage Arzan. _Ottawa, ON._ (poster and presentation)

\vspace{0.25em}

**Zaker, A.**, McRae, C., Renaud, J.M. (2021). Role of Adenosine Receptors on
mice skeletal muscle fatigue kinetics. _University of Ottawa UROP symposium._ Ottawa, ON. (abstract and poster).


######################################################################################################################################################################

\vspace{-0.5em}

# Software Projects and Repositories

**cut_run_pipeline_nix \href{https://github.com/ArvinZaker/cut_run_pipeline_nix}{\underline{\textsc{\faGithub}}} \hfill September 2024**

+ Collaborative project with Dr. Stefanelli lab.
+ Developed a pipeline for performing Cut&Run analysis and annotating the analysis outcomes.
+ Successfully stacked multiple technologies to create a robust pipeline.
+ Created a user-friendly guide for improving the utility of the pipeline for our collaborators.
+ Improved the reproducibility of existing pipelines by utilizing Nix programming language.

\vspace{0.25em}


**text_features \href{https://github.com/merlab/text_features}{\underline{\textsc{\faGithub}}} \hfill March 2024**

+ Compared and analyzed the impact of various feature selection methods in drug response prediction models.
+ Created scripts to be used in high-performance computing environment in the cloud and locally.
+ Created large-scale visualizations to communicate abstract findings to a wide audience.

\vspace{0.25em}


**SMC5-6_complex \href{https://github.com/merlab/SMC5-6_complex}{\underline{\textsc{\faGithub}}} \hfill September, 2023**

+ Collaborative project with Dr. D'Amours lab.
+ Aggregated pan-cancer dataset of over 65,000 patients from across 144 studies.
+ Analyzed the mutational effect of the SMC5/6 protein complex and its subunits.
+ Performed survival analysis to identify the SMC5/6-related risk factors to cancer prognosis.
+ Identified large-scale genomic instability associated with SMC5/6 mutation in patient transcriptome.

\vspace{0.25em}

**vian-clin-sci \href{https://github.com/ArvinZaker/vinas-clin-sci-2022}{\underline{\textsc{\faGithub}}} \hfill March 2023**

+ Reproduced the bioinformatics analysis performed by Viñas et al. (2020) 
\href{https://pubmed.ncbi.nlm.nih.gov/32662516/}{\underline{PMID: 32662516}}.
+ Developed a comprehensive report and delivered an in-depth presentation of the re-analysis, incorporating detailed visualizations and narrative summaries.
+ Optimized the analysis outcomes by integrating state-of-the-art bioinformatics tools to enhance data resolution and accuracy.

\vspace{0.25em}

**ATG16L1-MYPT1 \href{https://github.com/merlab/ATG16L1-MYPT1_Russell_Lab}{\underline{\textsc{\faGithub}}} \hfill November 2022**

+ Collaborative project with Dr. Ryan Russel lab.
+ Performed a differential expression analysis of VHL and RB knockout mice samples.
+ Identified the differentially expressed genes and pathways associated with knockouts.
+ Verified findings using novel literature text mining tools.

\vspace{0.25em}
